### ISSUE TITLE
eosio-cpp: Unknown command line argument '-g'.  Try: '/usr/local/eosio.wasmsdk//bin/eosio-cpp -help'

### ISSUE THREAD

**Issue by helperShang** [EOSIO/eosio.wasmsdk#40]
```
/Applications/CLion.app/Contents/bin/cmake/mac/bin/cmake --build /Volumes/mac/eos-api/work/hello/cmake-build-debug --target hello -- -j 2
Scanning dependencies of target hello
[ 50%] Building CXX object CMakeFiles/hello.dir/hello.cpp.o
eosio-cpp: Unknown command line argument '-g'.  Try: '/usr/local/eosio.wasmsdk//bin/eosio-cpp -help'
eosio-cpp: Did you mean '-C'?
make[3]: *** [CMakeFiles/hello.dir/hello.cpp.o] Error 1
make[2]: *** [CMakeFiles/hello.dir/all] Error 2
make[1]: *** [CMakeFiles/hello.dir/rule] Error 2
make: *** [hello] Error 2
```

**Comment by onigirisan** [comment]
The same problem as you, have you solved it?

**Comment by kesar** [comment]
+1

**Comment by larryk85** [comment]
It appears that you are trying to do a debug build.  Currently this is not supported, as it makes no sense for a smart contract (no debuggers and no specification for Dwarf).  Just reconfigure your cmake to Release.  @helperShang, @onigirisan and @kesar are you using Clion?

**Comment by kesar** [comment]
yes, im using clion. so far i wasnt able to get a proper integration w cmake

**Comment by larryk85** [comment]
Try this, change the line  <configuration PROFILE_NAME="Debug" CONFIG_NAME="Debug" /> . to 
<configuration PROFILE_NAME="Debug" CONFIG_NAME="Release" /> in .idea/workspace.xml

**Comment by kesar** [comment]
@larryk85 i get the one mentioned by @helperShang 👍

**Comment by larryk85** [comment]
Okay, remove the lines  
<component name="XDebuggerManager">
    <breakpoint-manager />
    <watches-manager />
  </component>

**Comment by kesar** [comment]
I dont have that in my .idea/* files

**Comment by GiggleAll** [comment]
Did you solve this problem. I didn't hava also this .idea/*files.

**Comment by GiggleAll** [comment]
I run eosio-cpp <path>/hello.cpp, it output "a.out", and -o can change the file name. maybe it can work.(I didn't have a try).    But macOsX + Clion still doesn't work：）

**Comment by GiggleAll** [comment]
@kesar Do you use eos to write smart contracts? Today, I compiled the source code of eos. It works very well with macOS+clion. Now, wasmsdk is not perfect, and eos is very convenient.

**Comment by larryk85** [comment]
Resolved by PR #48
